src/authInfo.ts

```diff
@@ -1099,27 +1104,33 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
   private async retrieveUsername(
     instanceUrl: string,
     accessToken: string,
-    orgId: string,
-    userId: string
-  ): Promise<Optional<string>> {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    orgIdToRemove?: string,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    userId?: string
+  ): Promise<Optional<UserInfo>> {
     // Make a REST call for the username directly.  Normally this is done via a connection
     // but we don't want to create circular dependencies or lots of snowflakes
     // within this file to support it.
-    const apiVersion = 'v42.0'; // hardcoding to v42.0 just for this call is okay.
     const instance = ensure(instanceUrl);
-    const url = `${instance}/services/data/${apiVersion}/sobjects/User/${userId}`;
+    const url = `${instance}/services/oauth2/userinfo`;
```

Comment:
Hit endpoint that returns the current user's information.

Reply:
I like this unversioned endpoint better.